lib/tariff_synchronizer/mailer.rb

```diff
@@ -27,12 +27,6 @@ def failed_download(exception, url)
       mail subject: "#{subject_prefix(:error)} Trade Tariff download failure"
     end
 
-    def failures_reminder(file_names)
```

Comment:
@rusllonrails @matthewford maybe we should keep mailer code in case reuse it in future

Reply:
@alexey-klimuk not sure, @matthewford what is your opinion?

Reply:
that's what git is for, its still here if we need it, can remove.